RELEASE_NOTES.md

```diff
@@ -1,5 +1,83 @@
+  via the REST API.
+* Add a `BiHashMap` to `Mesh` for a unique ID to the mesh ID.
+* Allow `BiHashMap` look-ups with elements that implement the `Borrow` trait.
+* Replace the `RecvError::InternalError` with a `Disconnected` error for accuracy.
+* Experimental feature “circuit-template” (new): Implement `CircuitTemplateManager`
+  with functionality to list and load available templates.
+* For all experimental Biome features:
+    - Remove Arc wrappings from Biome’s user store to allow mutable references of
+      the user store.
+    - Separate Biome models and schemas into their respective modules.
+    - Add a `/biome/verify` REST API endpoint to verify a user’s password.
+    - Update the Biome Rust API documentation to remove explicit references to
+      Rust features.
+    - Make the Biome user module private.
+    - Rename `biome::datastore` to `biome::migrations` and add module-level
+      documentation for biome migrations.
+    - Remove all uses of `super` imports in Biome.
+* Correct `json-web-tokens` feature guards.
+* Experimental feature “proposal-read”:
+    - Update the `ProposalStore::proposals` arguments to take a `ProposalFilter`
+      enum to allow the proposal store to filter returned proposals.
+    - Rename the `/admin/proposals` endpoint's `filter` query parameter to `management_type`.
+    - Add `member` filter query parameter to the GET `/admin/proposals/` REST API endpoint.
+
+### CLI:
+* Experimental feature “circuit”:
+    - Update the `splinter circuit create` command to generate a service ID automatically.
+    - Add a `comments` argument to the `splinter circuit create` command to optionally
+      provide comment for the proposed circuit.
+    -  Display comments in the output of the `splinter proposals` command.
+    - Update the `circuit-read` and `proposal-read` response deserializers to reflect
+      the REST API response objects, rather than the related internal structs.
+    - Add a `member` argument to the `splinter circuit proposals` command for
+      filtering proposals by the given member.
+    - Add `SPLINTER_REST_API_URL` environment variable to be used if `-U` or `--url`
+      is not specified.
+    - Remove the `required` specification from `splinter circuit list` and
+      `splinter circuit proposals` arguments that are not actually required.
+* Rename the `DEFAULT_ENDPOINT` constant to `DEFAULT_SPLINTER_REST_API_URL`.
+* Experimental feature “circuit-template” (new): Implement the experimental
+  `circuit-template` feature. This feature lets you use a template to create a
+   circuit with the new `template` subcommand for `splinter circuit create`.
+    - Add the `dry_run` flag to the `circuit create` command, which displays the
+      circuit definition without submitting the proposal.
+    - Add subcommands behind the `circuit-template` feature to display circuit templates:
+      `splinter circuit template list` and `splinter circuit template show`.
+
+### packaging:
```

Comment:
Should also mention that as a part of the `circuit-template` feature, `scabbard` template is packaged with the splinter CLI and the `gameroom` template is packaged with the gameroom daemon.

Reply:
Addressed.